server/world/world.go

```diff
@@ -1449,6 +1566,60 @@ func (w *World) tickEntities(tick int64) {
 	w.entitiesToTick = w.entitiesToTick[:0]
 }
 
+// SetRaining toggles raining depending on the raining argument.
+func (w *World) SetRaining(raining bool, x time.Duration) {
```

Comment:
Would maybe split this up into StartRaining(time.Duration) and StopRaining(). The duration argument isn't useful here. Same for thunder.

Reply:
I see, I can agree with that. I will do that then.

Reply:
Though, I think it is only best to do that for the exported methods since it would become a huge mess if we'd also do it for the unexported ones.

Reply:
Oh yeah I wouldn't do it for the unexported ones.